src/functions/utils/case-sensitive-video-id-mapper.js

```diff
@@ -0,0 +1,30 @@
+const getCaseSensitiveYoutubeVideoId = (videoId) => {
```

Comment:
we shouldn't have this content (ids) in git right?

Reply:
This is only generated when we do the prerender. Someone that use `netlify dev` for the first time might not have this .js file and then it will not work that's why I put it in git

Reply:
Ok since I used mapping in `.json` file I added the `.json` file to the .gitignore.

The `getCaseSensitiveYoutubeVideoId` method will return the videoId if it cannot find or parse the .json mapping